### PR TITLE
Add ansible learning paths

### PR DESCRIPTION
**Pull request by audgirka**
## Description

This PR adds ansible learning paths to data.json.
Once merged these will be visible in learning paths section
and used to show these in ansible plugin.

## Which issue(s) does this PR fix

- Fixes #?

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: packages/app/public/learning-paths/data.json [hunk 1]
```diff
@@ -272,5 +272,77 @@
...
     "label": "Using projects in Red Hat OpenShift Data Science",
     "url": "https://developers.redhat.com/learn/openshift-data-science/using-projects-red-hat-openshift-data-science"
+  },
+  {
+    "minutes": 30,
+    "level": "Beginner",
+    "type": "Ansible Learning path",
+    "description": "Learn more about the fundamental concepts of Ansible, an open source IT automation solution that automates provisioning, configuration management, application deployment, orchestration, and many other IT processes.",
+    "label": "Introduction to Ansible",
+    "url": "https://red.ht/aap-lp-ansible-basics"
```
Comment (Zaperex) [inline packages/app/public/learning-paths/data.json:282]: This link redirects to https://developers.redhat.com/products/ansible/overview

File: packages/app/public/learning-paths/data.json [hunk 2]
```diff
@@ -272,5 +272,77 @@
...
+  },
+  {
+    "hours": 1,
+    "level": "Beginner",
+    "type": "Ansible Learning path",
+    "description": "Do a deep-dive into the features of Ansible VS Code extension, the Red Hat recommended way of creating your Ansible projects, which brings syntax-highlighting, auto-completion, linting, and generative AI to your automation creation workflows.",
+    "label": "Getting started with the Ansible VS Code extension",
+    "url": "https://red.ht/aap-lp-vscode-essentials"
```
Comment (Zaperex) [inline packages/app/public/learning-paths/data.json:290]: This link also seems to no longer exist and redirects to https://developers.redhat.com/products/ansible/overview

File: packages/app/public/learning-paths/data.json [hunk 3]
```diff
@@ -272,5 +272,77 @@
...
+    "label": "YAML Essentials for Ansible",
+    "url": "https://red.ht/aap-lp-yaml-essentials"
+  },
+  {
+    "hours": 1,
+    "level": "Beginner",
+    "type": "Ansible Learning path",
+    "description": "Learn how to create Ansible playbooks, the blueprints for automation tasks executed across an inventory of nodes. Playbooks tell Ansible which tasks to perform on which devices.",
+    "label": "Getting started with Ansible playbooks",
+    "url": "https://red.ht/aap-lp-getting-started-playbooks"
```
Comment (Zaperex) [inline packages/app/public/learning-paths/data.json:306]: This link also redirects to https://developers.redhat.com/products/ansible/overview

File: packages/app/public/learning-paths/data.json [hunk 4]
```diff
@@ -272,5 +272,77 @@
...
+  },
+  {
+    "hours": 1,
+    "level": "Intermediate",
+    "type": "Ansible Learning path",
+    "description": "Learn more about Ansible Content Collections. Content Collections provide a format for organizing and sharing your Ansible content, such as modules, Playbooks, and documentation.",
+    "label": "Getting started with Content Collections",
+    "url": "https://red.ht/aap-lp-getting-started-collections"
```
Comment (Zaperex) [inline packages/app/public/learning-paths/data.json:314]: Redirects to https://developers.redhat.com/products/ansible/overview

**Review by Zaperex — COMMENTED** [review]
A bunch of the learning path links here no longer seem to be valid and instead redirect to  https://developers.redhat.com/products/ansible/overview

**Comment by audgirka** [comment]
@Zaperex  These are dynamic links and they will be updated by the stakeholders from the backend once the learning paths are live.

@ganeshrn ^^

**Review by Zaperex — APPROVED** [review]
If that's the case, LGTM

**Pull request merged**